news/_posts/2021-03-02-archived-modules.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Focusing on essentials modules"
+subtitle: "Get some details about the archived modules"
+date:   2021-03-02 14:00:00
+authors:  [ MarionFrancois ]
+icon: icon-puzzle
+tags: 
+ - modules
+ - news
+---
+
+The product team recently did a survey of 600 PrestaShop users to find out which modules were essential and which they used the least.
```

Comment:
How about starting the article citing the number of modules we identified. Then you can go on saying that resources are limited, and therefore some modules haven't received updates for years. You can then go on to the survey, and the conclusion following it.

Reply:
I think it's important to start by explaining our research approach with the survey. So readers can immediately understand that we did not do this randomly due to a lack of resources and that what matters first to us is their needs.

Reply:
I understand, but we don't understand why the survey was done in the first place and the cause-consequence relationship between the number of modules and the resources needed to maintain them.

To me, the process goes this way:

We have a _lot_ of modules in the project -> This requires a lot of resources we don't have (proof: many modules untouched for years) -> Let's find which ones are actually useful -> Survey -> Okay, let's abandon the ones that aren't useful and focus on the most important ones.

Furthermore, that's what actually happened 😄 

Otherwise people will read it like this:

PrestaShop wants people to buy stuff and hates open source -> find out excuse to get rid of perfectly good open source modules -> drop in some lazy excuse -> screw users out of greed